apps/preview/next/pages/showcases/Filters/Sizes.tsx

```diff
@@ -40,12 +40,13 @@ export default function FilterSizes() {
         </div>
       }
     >
-      <ul className="flex flex-wrap gap-4 px-1.5">
+      <ul className="grid grid-cols-5 gap-2">
         {chipSizes.map(({ id, label, value, isAvailable }) => (
           <li>
             <SfChip
               key={id}
               size="sm"
+              className="w-16"
```

Comment:
whole width of the container should be provided equally to all of the elements
```suggestion
              className="w-full"
```